Neptune.WebMvc/Views/FieldVisit/FieldVisitGridSpec.cs

```diff
@@ -81,6 +82,15 @@ public FieldVisitGridSpec(Person currentPerson, bool detailPage, LinkGenerator l
                 Add(FieldDefinitionType.Jurisdiction.ToGridHeaderString(), x => UrlTemplate.MakeHrefString(stormwaterJurisdictionDetailUrlTemplate.ParameterReplace(x.StormwaterJurisdictionID), x.OrganizationName), 140, DhtmlxGridColumnFilterType.SelectFilterHtmlStrict);
             }
 

```

Comment:
can you combine these within the same if since they are right next to each other?